docs-source/content/quickstart/cleanup.md

```diff
@@ -10,9 +10,18 @@ weight: 7
 
 1.	Remove the domain's Ingress by using `helm`:
```

Comment:
General comment:  This file has references to helm upgrade that don't show the 2.x and 3.x versions.

Reply:
Does the new syntax work on both 2.x and 3.x?

Reply:
upgrade works for both 2 and 3